assignments/callbacks.js

```diff
@@ -24,28 +24,47 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
+getLength(items, function(length) {
+  console.log(length);
+});
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[arr.length - 1]);
 }
-
+last(items, function(last) {
+  console.log(last);
+});
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x, y);
```

Comment:
looks like its asking you to add the numbers in the higher order function, and pass that result into the call back.
so just changing the return works
```
return cb(x + y);
```